### ISSUE TITLE
A way of including all examples by default

### ISSUE THREAD

**Issue by zikalino** [Azure/magic-module-specs#28]
Is there a way of including all autogenerated examples in the module by default from "examples" folder by default?
I think it doesn't make sense to require to include them by manually adding entries to **ansible.yml** and **terraform.yml** files.
If we don't want to include particular example, we should simply remove it from examples directory.

**Comment by JunyiYi** [comment]
No, there is no. We require the user to be carefully thinking about their test cases. So we require them to manually write all and include them in terraform.yaml or ansible.yaml.
